### PR TITLE
h2 title

### PR DESCRIPTION
**Pull request by otigreros**
add new CustomTextViewStyle to the CustomTextView

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/customtextview/readme.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 |  SUBTITLE_TYPE | 2 |
 |  SECTION_TITLE_TYPE | 3 |
 |  LINK_TYPE | 4 |
+|  h2_TITLE_TYPE | 5 |
```
Comment (EAlexisDuque) [inline app/src/main/java/co/condorlabs/customcomponents/customtextview/readme.md]: in capital letters?

Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/customtextview/readme.md]: are constants, so the documentation goes with the same type

Comment (otigreros) [reply app/src/main/java/co/condorlabs/customcomponents/customtextview/readme.md]: done

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt [hunk 2]
```diff
@@ -225,7 +225,7 @@ open class BaseEditTextFormField(context: Context, private val attrs: AttributeS
     }
 
     protected fun doesTextMatchWithRegex(candidateText: String?): Boolean {
-        if(candidateText == null ){
+        if (candidateText == null) {
```
Comment (diriarte-condor) [inline app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt]: use isNull()

**Pull request merged**